sound/soc/sof/sof-priv.h

```diff
@@ -290,6 +298,43 @@ struct snd_sof_dai {
 	struct list_head list;	/* list in sdev dai list */
 };
 
+/*
+ * in virtio iovec array:
+ *  iovec[0]: the ipc message data between vFE and vBE
+ *  iovec[1]: the ipc reply data between vFE and vBE
+ */
+#define SOF_VIRTIO_IPC_MSG 0
+#define SOF_VIRTIO_IPC_REPLY 1
+
+/* Virtio Frontend */
+struct sof_vfe {
+	struct sof_virtio_priv     *priv;
+	struct snd_sof_dev *sdev;
+
+	/* IPC cmd from frontend to backend */
+	struct virtqueue           *ipc_cmd_tx_vq;
+	/* IPC cmd reply from backend to frontend */
+	struct virtqueue           *ipc_cmd_rx_vq;
+	/* IPC notification from backend to frontend */
+	struct virtqueue           *ipc_not_rx_vq;
+	/* IPC notification reply from frontend to backend */
+	struct virtqueue           *ipc_not_tx_vq;
```

Comment:
indentation of variable names - some members use a tab and others a space in this struct.

Reply:
got it. I misunderstood your meaning last time.